### DISCUSSION TITLE
Meta sync [Dec 10]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4667] [Discord Threads]
> 🙋‍♂️ **jeanbonbeur**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### NocoDB will continue to work with your existing PG database. Additional meta information required is maintained in NocoDB's meta DB. Not detailed, but you will get a rough idea from here https://docs.nocodb.com/engineering/architecture   
####    
#### Ideally, once you connect to your external DB - all your tables and SQL views should get listed in NocoDB dashboard - along with paginated data. NocoDB doesn't create an additional replica of your DB & the data changes will always be in sync with your DB.    
####    
#### Any schema changes done outside NocoDB needs an explicit Sync. https://docs.nocodb.com/setup-and-usages/sync-schema   
#### So, if a table is deleted using PGAdmin, you will have to trigger explicit Sync. Tables will not be automatically deleted in the dashboard. For the issue that you are seeing, I would recommend to trigger an explicit meta-sync.  
#  
> 🙋‍♂️ **jeanbonbeur**  
#### OK, thks a lot. It's more clear! 🙂  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1039935650885017671)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
